org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/ProcessModalDialog.java

```diff
@@ -343,4 +349,12 @@ protected void onClientInfo() {
 			}
 		}
 	}
+
+	/**
+	 * Set Report Viewer
+	 * @param reportViewer
+	 */
+	public void setReportViewer(ZkReportViewer reportViewer) {
+		this.reportViewer = reportViewer;
+	}
 }	//	ProcessModalDialog
```

Comment:
Should use a mechanism similar to ProcessDialog instead of this ZkReportViewer only approach.